mne/preprocessing/nirs/nirs.py

```diff
@@ -69,7 +69,7 @@ def _check_channels_ordered(raw, freqs):
     """Check channels followed expected fNIRS format."""
     # Every second channel should be same SD pair
     # and have the specified light frequencies.
-    picks = _picks_to_idx(raw.info, 'fnirs_od')
+    picks = _picks_to_idx(raw.info, 'fnirs_od', exclude=[])
```

Comment:
At some stage of processing, do we need to do something about pairs of channels where only one is marked bad? Basically whatever processing step(s) actually combine the data from two channels should probably "spread the badness", i.e., if one of them is marked bad before the operation, the operation can do its thing, but both should be marked as bad afterward.

Reply:
That's a great idea. I will write two function. One that checks that the labeling of bads is consistent across pairs of optodes, something like `_fnirs_check_bads`. And another function that spreads bads if only a single optode frequency was marked bad, something like `_fnirs_spread_bads`. What do you think of this? I was thinking to not expose these functions as they seem advanced and the automated quality check functions like `scalp_coupling_index` should behave correctly and also mark the correct number of bads per optode.

Reply:
Can I do add those functions in another PR? This one seems like a bug fix, the new functions are new features.

Reply:
> Can I do add those functions in another PR? This one seems like a bug fix, the new functions are new features.

Ignore, I just did it here.